go.mod

```diff
@@ -31,108 +29,110 @@ require (
 	golang.org/x/crypto v0.15.0
 	golang.org/x/text v0.14.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.27.3
-	k8s.io/apiextensions-apiserver v0.27.3
-	k8s.io/apimachinery v0.27.3
-	k8s.io/cli-runtime v0.27.3
-	k8s.io/client-go v0.27.3
-	k8s.io/component-base v0.27.3
+	k8s.io/api v0.28.3
```

Comment:
We could probably change this to 0.28.4 now but lets do that afterwards.

Reply:
sgtm

Reply:
done